Client/Components/DashBoard.razor.cs

```diff
@@ -21,12 +21,14 @@ public partial class DashBoard
         
         DateOnly date = new DateOnly(2022, 1, 1);
         private IList<string> dataMetrics = new List<string>();
+        private IList<string> validMetrics = new List<string>();
```

Comment:
What is valid metrics?

Reply:
Valid metrics is the available datametrics. I use two variables for the data metrics as the actual "dataMetrics" variable is going to change. So to avoid having to call GetValidMetrics() every time I need to get all available metrics, I just save them to this "valid metrics" variable once. I agree that the name could be clearer.